components/ui/Gnb.vue

```diff
@@ -12,15 +12,15 @@
                 background-color="#212121"
                 text-color="#fff"
                 active-text-color="#ffd04b">
-                <el-menu-item index="guide" >덱 가이드</el-menu-item>
+                <el-menu-item index="guide">덱 가이드</el-menu-item>
 
-                <el-submenu
+                <el-submenu v-if="$store.getters['seasons']"
                     :hide-timeout="500"
                     popper-class="seasons-menu"
                     index="season">
-                    <template slot="title">시즌 {{ currentSeason }}</template>
-                    <el-menu-item v-for="(season, s_idx) in $store.state.sessionList" :index="season.num.toString()"
-                                  :key="s_idx">
+                    <template slot="title">{{ $store.state.seasonInfo.title }}</template>
```

Comment:
Codacy has a fix for the issue: [Replace `·$store.state.seasonInfo.title` with `⏎························$store.state.seasonInfo.title⏎···················`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
                    <template slot="title">{{
```